client/my-sites/current-site/index.jsx

```diff
@@ -113,7 +113,6 @@ const CurrentSite = React.createClass( {
 	},
 
 	previewSite: function( event ) {
-		analytics.ga.recordEvent( 'Sidebar', 'Clicked View Site' );
```

Comment:
@folletto I moved the GA event up to the container's `onPreviewSite` method to catch the sidebar button as well. If it's left here we get duplicated events on this click.

Most of the `/my-sites/sidebar` items do not record clicks. Page views are recorded on navigation.

On looking, I did find that the `plans` sidebar item tracks `calypso_upgrade_nudge_cta_click` on click, which does not appear correct.